docs/release/release_0_2_7.rst

```diff
@@ -0,0 +1,81 @@
+Announcement: napari 0.2.7
+==========================
+
+We're happy to announce the release of napari v0.2.7!
```

Comment:
Has this v always been here? I don't love it, compared to just 0.2.7, as above.

Reply:
```suggestion
We're happy to announce the release of napari 0.2.7!
```

Reply:
fixed